src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/ClanManager.java

```diff
@@ -660,6 +609,8 @@ public String getWeaponString(PlayerInventory inv) {
         return out;
     }
```

Comment:
There are a lot of same checking (`if (count > 0)`), should be refactored.

Reply:
Yes, but we are continuously updating the count variable instead of declaring different counts.

Reply:
Look over my idea with enum, there is one checking through the for loop.

Reply:
@Tomut0 please check my latest commit will this approach do instead of enum?

Reply:
I don't think so, you replaced the duplicated code from one place to another one.